packages/graphql-api/src/modules/accounts/schema/types.ts

```diff
@@ -1,6 +1,7 @@
 import gql from 'graphql-tag';
+import { AccountsModuleConfig } from '..';
 
-export default gql`
+export default ({ userAsInterface }: AccountsModuleConfig) => gql`
```

Comment:
Do you think the usage of setting up the `User` interface should be added to the documentation? A note about this in the readme and graphql docs would be useful I think.

Reply:
Oh sorry, I forgot to add this change in the docs. I can remove this if you want.
Otherwise, I can add this in the docs.

Reply:
I think it's useful, please add to the docs.

Reply:
I added some info to docs. Let me know if it is good.